test/unit/org/apache/cassandra/db/compaction/CompactionStrategyManagerPendingRepairTest.java

```diff
@@ -352,15 +355,6 @@ public void testFinalizedAndCompactionRace() throws IOException
         LocalSessionAccessor.finalizeUnsafe(repairID);
         LocalSession session = ARS.consistent.local.getSession(repairID);
         ARS.consistent.local.sessionCompleted(session);
-        Assert.assertTrue(hasPendingStrategiesFor(repairID));
```

Comment:
I think this still makes sense to check.

Reply:
added back